README.md

```diff
@@ -13,97 +13,129 @@ urlFragment: functions-quickstart-python-azd
++ [Azure Developer CLI (AZD)](https://learn.microsoft.com/azure/developer/azure-developer-cli/install-azd)
++ To use Visual Studio Code to run and debug locally:
+  + [Visual Studio Code](https://code.visualstudio.com/) 
+  + [Azure Functions extension](https://marketplace.visualstudio.com/items?itemName=ms-azuretools.vscode-azurefunctions)
 
 ### Get repo on your local machine
-Run the following GIT command to clone this repository to your local machine.
-```bash
-git clone https://github.com/Azure-Samples/functions-quickstart-python-azd.git
-```
 
-## Run on your local environment
+You can download this project template in one of these ways:
+
++ Use this `azd init` command from an empty local (root) folder:
+
+    ```azd
+    azd init --template functions-quickstart-python-http-azd
+    ```
+
++ Clone this repository locally using the `git clone` command:
+
+    ```bash
+    git clone https://github.com/Azure-Samples/functions-quickstart-python-azd.git
+    cd functions-quickstart-python-azd
+    ```
+
++ Fork this repository to your GitHub account and then clone locally using the `git clone` command:
+
+    ```bash
+    git clone https://github.com/<YOUR_ACCOUNT>/functions-quickstart-python-azd.git
+    cd functions-quickstart-python-azd
+    ```
+
+## Prepare your local environment
+
+Add a file named `local.settings.json` in the root of your project with the following contents:
 
-### Prepare your local environment
-1) Add a file named local.settings.json file to the root of the project with the following contents. This will allow you to run your function locally.
 ```json
 {
-  "IsEncrypted": false,
-  "Values": {
+    "IsEncrypted": false,
+    "Values": {
     "AzureWebJobsStorage": "UseDevelopmentStorage=true",
     "FUNCTIONS_WORKER_RUNTIME": "python"
-  }
+    }
 }
 ```
 
-### Create a virtual environment
-1) Open the terminal, navigate to the project folder, and run the following commands:
+## Create a virtual environment
+
+The way that you create your virtual environment depends on your operating system.
+Open the terminal, navigate to the project folder, and run these commands:
+
+### Linux/macOs
 
 ```bash
 python -m venv .venv
 source .venv/bin/activate
 ```
 
-### Using Functions CLI
-1) Open this folder in a new terminal and run the following commands:
+#### Windows
 
-```bash
-pip install -r requirements.txt
-func start
+```shell
+py -m venv .venv
+.venv\scripts\activate
 ```
 
-2) Test the HTTP GET trigger using the browser to open [http://localhost:7071/api/http_get](http://localhost:7071/api/http_get)
+## Run your app from the terminal
 
-3) Test the HTTP POST trigger in a new terminal window:
-```bash
-curl -i -X POST http://localhost:7071/api/http_post -H "Content-Type: text/json" -d '{"name":"yourname"}'
-```
+1. Run these commands in the virtual environment:
+
+    ```bash
+    pip install -r requirements.txt
```

Comment:
```suggestion
    pip3 install -r requirements.txt
```

pip3 and python3 work better multi plat.  pip and python only aliases work best on only Windows.